proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,62 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib command [OPTIONS]`
+
+## Options
```

Comment:
super nit: maybe switch the ordering of the "Commands" section and "Options" section to match the format in the usage section